src/loader.py

```diff
@@ -152,7 +156,6 @@ async def summarize_image_document(doc: ImageDocument, client):
     client = ollama.AsyncClient()
```

Comment:
I wanted to switch this to litellm as well, but they don't make it easy to send a local file for vision because of a dependency on requests https://github.com/BerriAI/litellm/blob/3a35a58859a145a4a568548316a1930340e7440a/litellm/llms/prompt_templates/factory.py#L624-L635